2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "Remote Mapping in Malawi: Supporting Field Operations"
+postID: 2023-06-02-Malawi-MSF-Cholera-and-Cyclone-Freddy-Response
```

Comment:
including the date in the `postID` will mean the URL looks like this:
https://www.missingmaps.org/blog/2023/06/02/2023-06-02-Malawi-MSF-Cholera-and-Cyclone-Freddy-Response/ (with the date in there twice)
suggest removing